ha/setup/ha_setup.py

```diff
@@ -198,7 +198,8 @@ def process(self):
                     cluster_create(cluster_name, nodelist)
                     Log.info(f"Created cluster: {cluster_name} successfully")
                     Log.info("Creating pacemaker resources")
-                    create_all_resources()
+                    s3_instances = Conf.get(self._index, f"cluster.{minion_name}.s3_instances")
```

Comment:
Let's read all Conf values before checking the cluster.
I think that it will throw the error if key is not present in Conf store. If that is the case, then we don't need default value anywhere in our code.

Also, lets validate that this value is a positive integer and then proceed otherwise throw the error and exit.

Reply:
agree

Reply:
Done